includes/class-omise-callback.php

```diff
@@ -0,0 +1,178 @@
+<?php
```

Comment:
I'm unconvinced that a singleton is the way to go here. I don't see any point to it

Reply:
@jonrandy I've mentioned here https://github.com/omise/omise-woocommerce/pull/167#discussion_r427031831

Reply:
Yes, I read that before. Still unconvinced

Reply:
Can you explain a bit on what part that doesn't make you buy the idea?

Reply:
(There are 3 approaches now, initiating object, singleton, or static.
I agreed that we don't need to initiate a new object every time and I have provided the reason why I go with singleton instead of state-less static class as there are some benefits that it can be achieved better than singleton. Maybe I missed some point that you see.)

Reply:
The class is just being used as a holder for some utility functions - a helper class. It has no state, and nothing that needs initialising before. Instantiating it in any way - be that as a singleton, or a regular class instance - just seems completely superfluous

Reply:
Ah yup, then that's the point I've considered. I think it would be a useful trade-off between state-less static class and extensibility for 3rd-party developers (as this project is an open source by its nature). 
It's understandable if you are not convinced by this approach but if it does not do any harm to the project, then let me see it through a bit. If at the end it does not be any useful, then let's change it to static. I'm still convinced with this approach as in WordPress and WooCommerce community nature.

Reply:
Such static class helpers can be subclassed and extended easily

Reply:
Would it be considered as a good code practice to extend a static class?